File: Java/MenuBancario.java
import java.util.Scanner;

public class MenuBancario {
  public static void main(String[] args) {

    try (Scanner scanner = new Scanner(System.in)) {
      int numero = 0;
      int saldo = 1000;

      while (true) {
        System.out.println("Esto es un Menu Bancario: ");
        System.out.println("1 - Extracto de cuenta: ");
        System.out.println("2 - Ingresa 100 en cuenta: ");
        System.out.println("3 - Reintegro 100 de cuenta: ");
        System.out.println("4 - Salir ");
        System.out.print("Elije Opcion: ");
        numero = scanner.nextInt();

        System.out.println("-------------------");
        if (numero == 1) {
          System.out.println("Tienes " + saldo + " Euros de Saldo.");
        } else if (numero == 2) {
          saldo += 100;
          System.out.println("Ingresando 100 en Cuenta, tienes " + saldo + " Euros");
        } else if (numero == 3) {
          saldo -= 100;
          System.out.println("Reintegro de 100 Euros, tienes " + saldo + " Euros");
        } else if (numero == 4){
          System.out.println("Hasta Pronto!");
          break;
        } else {
          System.out.println("Opcion no Valida!");
        }
        System.out.println("-------------------");
      }
      scanner.close();
    }
  }
}

File: C/MenuBancario.c
#include <stdio.h>
int main(int argc, char const *argv[]) {
  int saldo = 1000;
  int opcion = 0;

  while (1) {
    puts("Esto es un Menu Bancario: ");
    puts("1 - Extracto de cuenta: ");
    puts("2 - Ingresa 100 en cuenta: ");
    puts("3 - Reintegro 100 de cuenta: ");
    puts("4 - Salir ");
    printf("Elije Opcion: ");
    scanf("%d", &opcion);

    puts("-------------------");
    if (opcion == 1) {
      printf("Tienes %d Euros de Saldo.\n", saldo);
    } else if (opcion == 2) {
      saldo += 100;
      printf("Ingresando 100 en Cuenta, tienes %d Euros\n", saldo);
    } else if (opcion == 3) {
      saldo -= 100;
      printf("Reintegro de 100 Euros, tienes %d Euros\n", saldo);
    } else if (opcion == 4) {
      puts("Hasta Pronto!");
      break;
    } else { 
      puts("Opcion no Valida!");
    }
    puts("-------------------");
  }
  return 0;
}